doc/README.md

```diff
@@ -34,6 +34,45 @@ $ nix-build doc
 
 If the build succeeds, the manual will be in `./result/share/doc/nixpkgs/manual.html`.
 
+### Shell utilities
+
+In order to ease up your contribution workflow, consider loading the provided shell by using `nix-shell`.
+
+You can load it from the root of the nixpkgs tree with:
+
+```ShellSession
+$ cd /path/to/nixpkgs
+$ nix-shell doc
+```
+
+Alternatively, you can load it by first entering the doc folder:
+
+```ShellSession
+$ cd /path/to/nixpkgs/doc
+$ nix-shell
+```
+
+From there, all utilities provided by the `shell.nix` file will be available in your PATH.
```

Comment:
Which `shell.nix` file? The particular one in that directory doesn't even say what it provides because the source of truth lives somewhere else. Not sure the sentence is even needed...

```suggestion
```